one-ui/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.tsx

```diff
@@ -148,9 +149,11 @@ const CreateEditMappingDialog = (props) => {
 
     let status = await props.createMappingArtifact(dataPayload);
 
-    if (status === '200') {
+    if (status.code === 200) {
       props.setNewMap(false);
-    } else if (status === '400') {
+    } else if (status.code === 400) {
+      
+      setErrorMessage(status.message)
       setIsNameDuplicate(true);
```

Comment:
This is probably safe for now, but I wanted to note that the UI is making an assumption here - that a 400 status means that the name is a duplicate. That isn't guaranteed to be true, and it could certainly change in the future when some other validation rule is added.

Reply:
@rjrudin I'm reading this as that the UI is seeing a 400 error status and displaying the associated error message sent from the backend, there's not an assumption/interpretation of the type of error (e.g., duplicate name) on the UI's part other than it's a 400 status.